### ISSUE TITLE
Crash when mining cobble stairs with iron pick

### ISSUE THREAD

**Issue by Heptagon-ru** [anegostudios/VintageStory-Issues#190]
On PTS I was moving along the road from my base to Spawn and replacing old cobble stairs with new path ones. After about 10th time doing this game crashed. Maybe distant chunks were loading and something went wrong.

```
Running on 64 bit Windows with 16GB RAM 
22.09.2018 20:38:15: Critical error occurred
System.Exception: Error when trying to update vao, modeldata xyz/rgba/uv/indices sizes: 0/0/0/0 OpenGL threw an error: InvalidOperation
   в _JJC1FjbmfTUzh0UtoYqPtkQ7i0A._tyntGUuwzTfsPnIYHa2chttMTCe(String )
   в _JJC1FjbmfTUzh0UtoYqPtkQ7i0A._8nQcTCUPRUI0ro0qeEWI0pZIatZ(MeshRef , MeshData )
   в _xTGBclAWxbKKxWCtPdMnB2Yq52H._snVL8Fs24Dt5J9qHxfGfJiyOddj(Int64 , IWorldAccessor )
   в _6ZYP4I8bZ3OFriU531TV3qO8wFL._l7kaqJUlzfMJgZoOz6vTd6aRqLF(Single )
   в _6ZYP4I8bZ3OFriU531TV3qO8wFL._t0XCKgOaOdNHjvFkA1bdjycB0p7(Single )
   в _ytZDA9e6dLzoDc8jRQCHk41FfTJ._zRUUMHTj3pckyvW1yTYyJwWBUGA(Single )
   в _mS743LsjN0xGpTEpLeV0WHGWfhg._Sb6TikVoKiDoe8LxrR9Jp39rlEH(Single )
   в _mS743LsjN0xGpTEpLeV0WHGWfhg._NhrisRdCCX0qGxlbFs79h2W4fAf(Single )
   в _JJC1FjbmfTUzh0UtoYqPtkQ7i0A._NQXJsTI8WFKiWVTwzNdb1quCL0Z(Object , FrameEventArgs )
   в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   в OpenTK.GameWindow.DispatchRenderFrame()
   в OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   в _tUQ0wd7Yg3zJ0Y8RPnU7lSmMkkH._nk1CWne996bqayKRw7SThbyR57g(_QK4sXQ3qxOSMsnq3xWa64KQNllF , String[] )
   в _zwG29E5b0OAErM97rzrfge97vBl._nk1CWne996bqayKRw7SThbyR57g(ThreadStart )
-------------------------------

Event Log, last 3 entries
==================================
{ TimeGenerated = 01.08.2018 21:35:40, Site = , Source = .NET Runtime, Message = Приложение: Vintagestory.exe
Версия платформы: v4.0.30319
Описание. Процесс был завершен из-за необработанного исключения.
Сведения об исключении: System.AccessViolationException
Стек:
   в _0BaCPtLx09PLrj4nN8pJ00sx9sh._UCwURescfEXeSZiUCAIdRRMa5Qu(Vintagestory.API.Client.MeshRef)
   в Vintagestory.GameContent.EntityShapeRenderer.DoRender3DOpaqueBatched(Single, Boolean)
   в _ySDRGn9cgMcenfGNDL0Ss9jXpCS._wHguJRTGalMlDBQw1pPBDbxHIUH(Single)
   в _A3hV2ldZ9pExwf59SucThc80F1c._Zz9Dy13BfXlwxJlH1CzCRDKOvGC(Vintagestory.API.Client.EnumRenderStage, Single)
   в _HiyDgs3NWoAa70yqZHUt1zzZZXN._RdScbfa2135qeHIZqghI9hIB2Gk(Single)
   в _HiyDgs3NWoAa70yqZHUt1zzZZXN._QghPrYe5USBJ5yrVCGnfgjNmXmt(Single)
   в _BkqKaSEudlzB8KRTQUEpovwODY._BncU0iixMqwU04kUJc7xnW8rdBd(Single)
   в _W2TozBCnLy2eVyDjn9CbYGcuyJb._jaTF7B02DACBcanq4RJYUIkPxNd(Single)
   в _W2TozBCnLy2eVyDjn9CbYGcuyJb._uxFeJOFLQnnUbfHiZk3D1dwL7NE(Single)
   в _0BaCPtLx09PLrj4nN8pJ00sx9sh._xrjEPjoBibJlZ5u5rJcVZvXvcmL(System.Object, OpenTK.FrameEventArgs)
   в System.EventHandler`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Invoke(System.Object, System.__Canon)
   в OpenTK.GameWindow.RaiseRenderFrame(Double, Double ByRef)
   в OpenTK.GameWindow.DispatchRenderFrame()
   в OpenTK.GameWindow.Run(Double, Double)
   в _PkglWYgXxKoeSWDNiPQDOtOzd8A._WwcLROIGCSsEdcyWiCRVwgDfGLe(_uxEboymA8xOTitFU1L0Xt1f4edY, System.String[])
   в _DLldCwzmWxAc7YkftgyEl6oNBYAA._WwcLROIGCSsEdcyWiCRVwgDfGLe(System.Threading.ThreadStart)
   в _PkglWYgXxKoeSWDNiPQDOtOzd8A._b2tO4qIHlm0Bno88k5FFEXcFADDA(System.String[])
 }
{ TimeGenerated = 01.08.2018 21:30:55, Site = , Source = .NET Runtime, Message = Приложение: Vintagestory.exe
Версия платформы: v4.0.30319
Описание. Процесс был завершен из-за необработанного исключения.
Сведения об исключении: код исключения c0000005, адрес исключения 00000000597D0DC0
 }
{ TimeGenerated = 01.08.2018 21:22:51, Site = , Source = .NET Runtime, Message = Приложение: Vintagestory.exe
Версия платформы: v4.0.30319
Описание. Процесс был завершен из-за необработанного исключения.
Сведения об исключении: System.AccessViolationException
Стек:
   в _0BaCPtLx09PLrj4nN8pJ00sx9sh._UCwURescfEXeSZiUCAIdRRMa5Qu(Vintagestory.API.Client.MeshRef)
   в Vintagestory.GameContent.EntityShapeRenderer.DoRender3DOpaqueBatched(Single, Boolean)
   в _ySDRGn9cgMcenfGNDL0Ss9jXpCS._wHguJRTGalMlDBQw1pPBDbxHIUH(Single)
   в _A3hV2ldZ9pExwf59SucThc80F1c._Zz9Dy13BfXlwxJlH1CzCRDKOvGC(Vintagestory.API.Client.EnumRenderStage, Single)
   в _HiyDgs3NWoAa70yqZHUt1zzZZXN._RdScbfa2135qeHIZqghI9hIB2Gk(Single)
   в _HiyDgs3NWoAa70yqZHUt1zzZZXN._QghPrYe5USBJ5yrVCGnfgjNmXmt(Single)
   в _BkqKaSEudlzB8KRTQUEpovwODY._BncU0iixMqwU04kUJc7xnW8rdBd(Single)
   в _W2TozBCnLy2eVyDjn9CbYGcuyJb._jaTF7B02DACBcanq4RJYUIkPxNd(Single)
   в _W2TozBCnLy2eVyDjn9CbYGcuyJb._uxFeJOFLQnnUbfHiZk3D1dwL7NE(Single)
   в _0BaCPtLx09PLrj4nN8pJ00sx9sh._xrjEPjoBibJlZ5u5rJcVZvXvcmL(System.Object, OpenTK.FrameEventArgs)
   в System.EventHandler`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Invoke(System.Object, System.__Canon)
   в OpenTK.GameWindow.RaiseRenderFrame(Double, Double ByRef)
   в OpenTK.GameWindow.DispatchRenderFrame()
   в OpenTK.GameWindow.Run(Double, Double)
   в _PkglWYgXxKoeSWDNiPQDOtOzd8A._WwcLROIGCSsEdcyWiCRVwgDfGLe(_uxEboymA8xOTitFU1L0Xt1f4edY, System.String[])
   в _DLldCwzmWxAc7YkftgyEl6oNBYAA._WwcLROIGCSsEdcyWiCRVwgDfGLe(System.Threading.ThreadStart)
   в _PkglWYgXxKoeSWDNiPQDOtOzd8A._b2tO4qIHlm0Bno88k5FFEXcFADDA(System.String[])
 }
```

**Comment by copygirl** [comment]
By PTS are you talking about the semi-official server? Is it 1.7 or 1.6.11?

**Comment by tyronx** [comment]
@Heptagon-ru i see a crash in your event log from 1st august. Any memories what happened there?

**Comment by tyronx** [comment]
Most likely fixed in v1.7.8. Please reopen otherwise.
